chat-commands.js

```diff
@@ -2468,18 +2469,28 @@ exports.commands = {
 				Chat.uncacheTree('./tournaments');
 				global.Tournaments = require('./tournaments');
 				Tournaments.tournaments = runningTournaments;
-
+				Chat.notify(rooms, `${prefix}${user.name} used /hotpatch ${target}`);
```

Comment:
Simple solution is just to not notify if it gets to an `else` condition.

This might also be split into a different commit/pullreq.